lib/ansible/modules/windows/win_cert_stat.py

```diff
@@ -0,0 +1,235 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2016, Ansible, inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# this is a windows documentation stub.  actual code lives in the .ps1
+# file of the same name
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: win_cert_stat
```

Comment:
This should be called `win_certificate_stat` maybe even `win_certificate_info` to better align with the `_info` modules we seem to be headed towards. Doing the `_info` one is preferable as we can include functionality to enumerate all certificates in a store instead of just the 1 a user wants.